public/resources/core/js/controllers/product-order.controller.js

```diff
@@ -295,6 +303,9 @@
                 apiInfo.orderItem[i].billingAccount = [vm.billingAccount.serialize()];
             }
 
+            apiInfo.orderDate = new Date();
+            apiInfo.notificationContact = vm.billingAccount.getEmailAddress().emailAddress.toString();
```

Comment:
This is not correct, the result of getEmailAddress does not have an emailAddress field. Instead:
vm.billingAccount.getEmailAddress().medium.emailAddress